src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,56 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Stream;
+import mate.academy.dao.BookDao;
+import mate.academy.dao.BookDaoImpl;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final List<String> titles;
+    private static final ThreadLocalRandom random = ThreadLocalRandom.current();
+    private static final int MIN_PRICE = 100;
+    private static final int MAX_PRICE = 1000;
+
+    static {
+        titles = List.of(
+                "Clean Code", "Effective Java", "Head First Java",
+                "Thinking in Java", "Core Java", "Head First Design Patterns"
+        );
+    }
```

Comment:
this setup is too complex

I would simply create the books inside of the main method like:
```
books = List.of(
   new Book(...),
   new Book(...),
  ...
);
```

no need to use Stream, ThreadLocalRandom etc - keep things simple :) (remember about KISS principle)